web/src/settings_emoji.ts

```diff
@@ -214,7 +214,8 @@ function show_modal(): void {
     const html_body = render_add_emoji({});
 
     function add_custom_emoji(): void {
-        dialog_widget.show_dialog_spinner();
+        const $button = $("#dialog_widget_modal .modal__btn");
+        loading.show_spinner($button);
```

Comment:
Since, `settings_emoji` is already a dialog_widget, the show_spinner method is already being called over at https://github.com/ecxtacy/zulip/blob/ba201901c4a37600fd90951a2a19712bccc5e945/web/src/dialog_widget.ts#L180.